src/test/scala/is/hail/variant/GenomeReferenceSuite.scala

```diff
@@ -173,4 +173,58 @@ class GenomeReferenceSuite extends SparkSuite {
     val mapping = Map("23" -> "foo")
     TestUtils.interceptFatal("do not have remapped contigs in the reference genome")(GenomeReference.GRCh37.validateContigRemap(mapping))
   }
+
+  @Test def testComparisonOps() {
+    val gr = GenomeReference.GRCh37
```

Comment:
This happens in a couple tests, it seems to force the tests to assume the default reference is grch37. Should we instead grab the default reference? (which I assume is in some variable somewhere?)

Reply:
I purposefully set it to GRCh37 because when we eventually change the default to GRCh38, all of the tests will break. For example, "3" will become "chr3" in GRCh38.

Reply:
👍